static/app/data/platforms.tsx

```diff
@@ -1,108 +1,689 @@
-import integrationDocsPlatforms from 'integration-docs-platforms';
```

Comment:
Let's make sure we don't forget to cleanup all the things that handle downloading this.

* Webpack configuration that sets up this `integration-docs-platforms` alias
* Webpack plugin to download
* Python implementation to download

Reply:
@priscilawebdev is it this issue? https://github.com/getsentry/sentry/issues/53861
I also created a task for us to not forget about @evanpurkhiser 's items.
https://github.com/getsentry/sentry/issues/54835

Reply:
yes @matejminar before we can completely remove everything related to the `sentry-docs` fetch, we have to migrate the wizard docs for profiling and replay